wgsl/index.bs

```diff
@@ -3542,7 +3536,9 @@ the test name.
 * v-0026: The case selector values must have the same type as the selector expression.
 * v-0027: A literal value must not appear more than once in the case selectors for a switch statement.
 * v-0028: A fallthrough statement must not appear as the last statement in last clause of a switch.
-
+* v-0029: Return must come last in its block.
+* v-0030: The storage type of a struct containing a runtime-sized array must be a of storage class 
```

Comment:
v-0030 isn't quite right.   A struct type doesn't have a storage type or store type. A variable has a store type.

I suggest:

> If a runtime-sized array appears inside the store type for a variable, then its appearance must be as the last member of the structure type that is the store type for a storage buffer variable.  A runtime-sized array must not be the store type of a variable.

Reply:
Thanks for bringing this up, I created a separated PR to continue our discussions. https://github.com/gpuweb/gpuweb/pull/1252